src/components/publishing/display/canvas/container.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react"
+import sizeMe from "react-sizeme"
+import styled, { StyledFunction } from "styled-components"
+import { crop } from "../../../../utils/resizer"
+import { pMedia } from "../../../helpers"
+import { sizeMeRefreshRate } from "../../constants"
+import Slideshow from "./slideshow"
+import CanvasText from "./text"
```

Comment:
Ah I see now. Maybe this is more a preference thing, but if you are exporting the component as `DisplayCanvasText`, it might be helpful to use the same name when using the component in here as well. I'd also say I tend to make filenames consistent with the component name, so naming it something like `display_canvas_text.tsx` might be clearer.